arch/xtensa/src/esp32/esp32_efuse_utils.c

```diff
@@ -0,0 +1,484 @@
+/****************************************************************************
+ * arch/xtensa/src/esp32/esp32_efuse_utils.c
```

Comment:
What's the difference between this file and esp32_efuse.c?
As far as I went, all the functions are present in both files.

Reply:
Good catch, this file is reminiscent from old efuse driver that was integrated months ago